docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -11,19 +11,10 @@ To make this more formal, here is the rounding function that is used:
 
 [source,java]
 --------------------------------------------------
-rem = value % interval
-if (rem < 0) {
-    rem += interval
-}
-bucket_key = value - rem
+bucket_key = Math.floor((value - offset) / interval) * interval + offset
 --------------------------------------------------
 
-From the rounding function above it can be seen that the intervals themselves **must** be integers.
-
-WARNING: Currently, values are cast to integers before being bucketed, which
-might cause negative floating-point values to fall into the wrong bucket. For
-instance, `-4.5` with an interval of `2` would be cast to `-4`, and so would
-end up in the `-4 <= val < -2` bucket instead of the `-6 <= val < -4` bucket.
+The `interval` must be a positive decimal, while the `offset` must be a decimal in `[0 - interval[`.
```

Comment:
I wanted to denote the fact that the bound is half open, would you do it otherwise?